src/AppInstallerCLICore/ExecutionContext.h

```diff
@@ -102,6 +105,8 @@ namespace AppInstaller::CLI::Execution
         }
 
         virtual void SetExecutionStage(Workflow::ExecutionStage stage, bool);
+        
+        void SetThreadLocalThreadGlobals();

```

Comment:
This will need to return the internal RAII type that ThreadGlobals will return from its SetActive type function.